src/containers/Home.jsx

```diff
@@ -101,5 +118,26 @@ const mapQueriesToProps = () => ({
   }
 })
 
+const mapMutationsToProps = () => ({
+  // createCampaign: (campaign) => ({
```

Comment:
probably need to add this back in so the rest of the campaign admin page works again